yellowbrick/text/dispersion.py

```diff
@@ -103,6 +119,11 @@ def draw(self, points, **kwargs):
         kwargs: generic keyword arguments.
         """
         
+        # Define boundaries with a solid vertical line
+        if self.set_boundaries:
+            for xcoords in self.boundaries_:
+                self.ax.axvline(x=xcoords, color='lightgray', alpha=0.75,
```

Comment:
I like that you're trying to mute the document boundaries so there is a contrast between the words and the annotations - very nice! Perhaps setting `linestyle='--'` or `':'` would also help?